frontend/src/views/main/project/SuiteTestExecutionList.vue

```diff
@@ -1,37 +1,37 @@
 <template>
   <v-list-item-group>
-      <template v-for="({result, suiteTest}) in props.tests">
-          <v-divider/>
-          <v-list-item :value="result" @click="testInfo(suiteTest)">
-              <v-list-item-icon>
-                  <v-icon :color="getColor(result)" size="40">{{
-                      getIcon(result)
-                      }}
-                  </v-icon>
-              </v-list-item-icon>
-              <v-list-item-content>
-                  <v-list-item-title>
-                      <div class="d-flex justify-space-between">
-                          <span>{{ getTestName(suiteTest.test) }}</span>
-                          <div>
-                              <v-btn
-                                      v-if="result !== undefined && !result.passed"
-                                      text
-                                      icon
-                                      color="green"
-                                      disabled
-                              >
-                                  <v-icon>mdi-bug</v-icon>
-                              </v-btn>
-                              <v-btn
-                                      v-if="!compact"
-                                      text
-                                      icon
-                                      color="error"
-                                      @click.stop="removeTest(suiteTest)"
-                              >
-                                  <v-icon>delete</v-icon>
-                  </v-btn>
+    <template v-for="({result, suiteTest}) in props.tests">
+      <v-divider/>
+      <v-list-item :value="result" @click="testInfo(suiteTest)">
+        <v-list-item-icon>
+          <v-icon :color="getColor(result)" size="40">{{
+              getIcon(result)
+            }}
+          </v-icon>
+        </v-list-item-icon>
+        <v-list-item-content>
+          <v-list-item-title>
+            <div class="d-flex justify-space-between">
+              <span>{{ getTestName(suiteTest.test) }}</span>
+              <div>
+                <v-btn
+                    v-if="result !== undefined && !result.passed"
+                    text
+                    color="green"
+                    @click.stop="debugTest(result, suiteTest)"
+                >
+                  <v-icon>mdi-bug</v-icon>
+                  Debug
```

Comment:
as we discussed, you should only be able to debug a test that has at least 1 model and 1 dataset as inputs. For therest of the cases we should disable the buttong with a tooltip explaining why it's disabled

Reply:
data drift tests do not need a model @Googleton @andreybavt